examples/fuse_examples/imaging/classification/mnist/run_mnist_custom_pl_imp.py

```diff
@@ -0,0 +1,377 @@
+"""
+(C) Copyright 2021 IBM Corp.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Created on June 30, 2021
+
+"""
+
+"""
+MNIST classfier implementation that demonstrate end to end training, inference and evaluation using FuseMedML
+This example shows how to directly train a model using custom (your own) pytorch lightning module implementation 
+"""
+
+
+import copy
+import logging
+import os
+from typing import Any, Dict, List, OrderedDict, Sequence
+from fuse.utils.file_io.file_io import create_dir, save_dataframe
+from fuse.utils.ndict import NDict
+
+import torch
+import torch.nn.functional as F
+import torch.optim as optim
+import pytorch_lightning as pl 
+from torch.utils.data.dataloader import DataLoader
+
+from fuse.eval.evaluator import EvaluatorDefault 
+from fuse.eval.metrics.classification.metrics_thresholding_common import MetricApplyThresholds
+from fuse.eval.metrics.classification.metrics_classification_common import MetricAccuracy, MetricAUCROC, MetricROCCurve
+
+from fuse.data.utils.samplers import BatchSamplerDefault
+from fuse.data.utils.collates import CollateDefault
+
+from fuse.dl.losses.loss_default import LossDefault
+from fuse.dl.models.model_wrapper import ModelWrapSeqToDict
+from fuse.dl.lightning.pl_funcs import *
```

Comment:
maybe instead it's better to do:
```import fuse.dl.lightning.pl_funcs as pl_funcs```
and then do pl_funcs.blah
pl_funcs.blah2
in the code?